apps/api/v2/package.json

```diff
@@ -25,7 +25,7 @@
   },
   "dependencies": {
     "@calcom/platform-constants": "*",
-    "@calcom/platform-libraries": "0.0.2",
```

Comment:
use npm alias to be able to use specific versions of platform libraries for specific versions of api endpoints